DependencyInjection/Configuration.php

```diff
@@ -107,7 +107,11 @@ public function getConfigTreeBuilder()
                                 return $v;
                             })
                         ->end()
-                        ->defaultValue(class_exists('Doctrine\ORM\EntityManager'))->end()
+                        ->defaultValue(class_exists('Doctrine\ORM\EntityManager'))
+                    ->end()
+                    ->scalarNode('service_naming_strategy')
```

Comment:
Just make sure to register your bundle after this one; this should be fine then. 

I'd prefer to keep this as small as possible, i.e. no additional options or classes.